cmd/monaco/account/download.go

```diff
@@ -0,0 +1,80 @@
+/*
+ * @license
+ * Copyright 2023 Dynatrace LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package account
+
+import (
+	"fmt"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/cmd/monaco/cmdutils"
+	"github.com/dynatrace/dynatrace-configuration-as-code/v2/cmd/monaco/completion"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"log"
+)
+
+type downloadOpts struct {
+	auth
+	manifestName   string
+	accountName    string
+	projectName    string
+	accountUUID    string
+	outputFolder   string
+	forceOverwrite bool
+}
+
+type auth struct {
+	clientID, clientSecret string
+}
+
+func downloadCommand(fs afero.Fs) *cobra.Command {
+	opts := downloadOpts{}
+
+	cmd := &cobra.Command{
+		Use:               "download [flags]",
+		Short:             "Download account management resources",
+		Example:           "monaco account download --manifest manifest.yaml --account <account-name-defined-in-manifest> --project <project-defined-in-manifest>",
+		ValidArgsFunction: completion.SingleArgumentManifestFileCompletion,
+		PreRun:            cmdutils.SilenceUsageCommand(),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			cmd.SilenceUsage = true
+			return download(fs, opts)
+		},
+	}
+
+	cmd.Flags().StringVarP(&opts.manifestName, "manifest", "m", "manifest.yaml", "Name (and the path) to the manifest file. Defaults to 'manifest.yaml'")
+	cmd.Flags().StringVarP(&opts.accountName, "account", "a", "", "Account name defined in the manifest to download from")
+	cmd.Flags().StringVarP(&opts.projectName, "project", "p", "", "Project name defined in the manifest")
+	cmd.Flags().StringVarP(&opts.accountUUID, "uuid", "u", "", "Account uuid to use. Required when not using the '--manifest' flag")
+	cmd.Flags().StringVar(&opts.clientID, "oauth-client-id", "", "OAuth client ID environment variable. Required when using the '--uuid' flag")
+	cmd.Flags().StringVar(&opts.clientSecret, "oauth-client-secret", "", "oauth client secret environment variable. Required when using the '--uuid' flag")
```

Comment:
```suggestion
	cmd.Flags().StringVar(&opts.clientSecret, "oauth-client-secret", "", "OAuth client secret environment variable. Required when using the '--uuid' flag")
```

Consistency w/ line above